packages/components/src/presentation-components/tree/controlled/UseHierarchyLevelFiltering.ts

```diff
@@ -63,5 +75,5 @@ function applyHierarchyLevelFilter(nodeLoader: ITreeNodeLoader, modelSource: Tre
   if (updatedNode === undefined || !updatedNode.isExpanded || updatedNode.numChildren !== undefined) {
     return;
   }
-  nodeLoader.loadNode(updatedNode, 0).subscribe();
+  return nodeLoader.loadNode(updatedNode, 0).subscribe({ next: () => onComplete(nodeId) });
```

Comment:
should we use `complete` and `error` rather than `next`?

Reply:
I believe we also need to call `onComplete(nodeId)` in case of an error

Reply:
fixed